modules/wri_person/wri_person.install

```diff
@@ -93,3 +93,29 @@ function wri_person_create_person_group() {
     'weight' => 1,
   ])->save();
 }
+
+/**
+ * Update the person content type to add fields and field group.
+ */
+function wri_person_update_10001() {
+  // Add the fields.
+  \Drupal::service('distro_helper.updates')->installConfig('field.storage.node.field_alt_first_names', 'wri_person', 'install', 'TRUE');
+  \Drupal::service('distro_helper.updates')->installConfig('field.field.node.person.field_alt_first_names', 'wri_person', 'install', 'TRUE');
+  \Drupal::service('distro_helper.updates')->installConfig('field.storage.node.field_alt_last_names', 'wri_person', 'install', 'TRUE');
+  \Drupal::service('distro_helper.updates')->installConfig('field.field.node.person.field_alt_last_names', 'wri_person', 'install', 'TRUE');
+
+  // Updating the Person form/view displays.
+  \Drupal::service('distro_helper.updates')->updateConfig('core.entity_form_display.node.person.default', [
+    'third_party_settings#field_group#group_more#group_aliases',
```

Comment:
I think you don't need `group_more` in here? That should fix the missing Alias tab.